Dockerfile

```
@@ -0,0 +1,31 @@
+# Python3 Debian image
+FROM python:3
+LABEL maintainer="Yichi Zhang <ichicho@keio.jp>"
+
+WORKDIR /root
+
+# Basic packages
+RUN apt update && \
+    apt install -y --no-install-recommends \
+                curl \
+                unzip \
+                chromium && \
```

Is python:3 an official Debian/Ubuntu image?
if it's not, `apt-get clean` keeps the image size smaller. doesn't it?
>Official Debian and Ubuntu images automatically run apt-get clean, so explicit invocation is not required.
https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run
 
```suggestion
                chromium && \
                apt-get clean && \
```

---

The base image is supposed to be an official Debian release(I used python:3.9.0-buster for testing). I will add a Debian tag for the base Python3 image. Therefore,  I guess `apt-get clean` isn't necessary and `rm -rf /var/lib/apt/lists/*` is enough. If I'm wrong, please feel free to correct me.

---

The good news is we confirmed nagias can be run in docker. However, the image size is 1.3GB now. I will try to switch base image to alpine to make the whole image lighter.

---

Sorry, I don't know the underlying meaning of what documents say.
I have dug the docker code, but I couldn't figure out how apt-clean is called. (Notice: I'm a Docker beginner)
I simply thought the documents literally referring to Ubuntu/Debian package only.

Alpine is better than Debian:+1:

Memo:
`python:3` use Debian package
https://hub.docker.com/_/python
https://hub.docker.com/layers/python/library/python/3/images/sha256-db968378628299a9164f0ac38bf46d257ace7365e31b2ded3a7e9f3eed4ba497?context=explore
https://github.com/docker-library/python/

---

For Ubuntu/Debian, just run `rm -rf /var/lib/apt/lists/*` after every `apt-get install`(in same `RUN` command). `apt-get clean` will be automatically conducted in official image.
For other Linux distribution image, to clear cache, command should be adjusted.

>Memo:
>`python:3` use Debian package

This is my bad. `python:3` is ambiguous. Using a detailed tag is better.